packages/poi-preset-karma/index.js

```diff
@@ -102,6 +109,15 @@ module.exports = (options = {}) => {
         singleRun: !watch
       }
 
+      if (isTypeScript) {
+        defaultConfig.karmaTypescriptConfig = {
+          tsconfig: './tsconfig.json',
+          compilerOptions: {
+            module: 'commonjs'
+          }
+        }
+      }
```

Comment:
is this necessary?

Reply:
Hah, I asked the same question. From what I can tell form the docs it is the default for TS code. But it also works without. 

https://github.com/egoist/poi/issues/264#issuecomment-340182770

Reply:
Yes it is needed in order to be able to use ES6 import

Reply:
You can run the tests without but you'll get errors